bump.sh

```diff
@@ -0,0 +1,113 @@
+
+# Update layer/Dockerfile
+echo "Updating layer/Dockerfile..."
+sed -i "" "s/KUBECTL_VERSION=1.$PREV_KUBECTL_VERSION\.[0-9]/KUBECTL_VERSION=1.$1.0/" layer/Dockerfile   # Update KubeCtl version
+sed -i "" "s/HELM_VERSION=3.$PREV_HELM_VERSION\.[0-9]/HELM_VERSION=3.$2.0/" layer/Dockerfile          # Update Helm version
+
+# Update SPEC_VERSION in .projenrc.js
+echo "Updating .projenrc.js..."
+sed -i "" "s/SPEC_VERSION = '$PREV_KUBECTL_VERSION'/SPEC_VERSION = '$1'/" .projenrc.ts
+
+# Update Kubectl Lambda layer in src/kubectl-layer.ts
+echo "Updating src/kubectl-layer.ts..."
+sed -i "" "s/KubectlV${PREV_KUBECTL_VERSION}Layer/KubectlV${1}Layer/g" src/kubectl-layer.ts # Replace class name
+sed -i "" "s/kubectl 1.$PREV_KUBECTL_VERSION.[0-9]/kubectl 1.$1.0/" src/kubectl-layer.ts    # Update KubeCtl version in description
+sed -i "" "s/helm 3.$PREV_HELM_VERSION.[0-9]/helm 3.$2.0/" src/kubectl-layer.ts             # Update Helm version in description
+
+# Update test/kubectl-layer.test.ts and test/kubectl-layer.integ.ts with new construct's name
+echo "Updating test files..."
+sed -i "" "s/KubectlV${PREV_KUBECTL_VERSION}Layer/KubectlV${1}Layer/g" test/kubectl-layer.test.ts   # Replace class name in unit test
+sed -i "" "s/KubectlV${PREV_KUBECTL_VERSION}Layer/KubectlV${1}Layer/g" test/kubectl-layer.integ.ts  # Replace class name in integ test
+sed -i "" "s/kubectl 1.$PREV_KUBECTL_VERSION.[0-9]/kubectl 1.$1.0/" test/kubectl-layer.test.ts      # Update KubeCtl version in description
+sed -i "" "s/helm 3.$PREV_HELM_VERSION.[0-9]/helm 3.$2.0/" test/kubectl-layer.test.ts               # Update Helm version in description
+
+echo "All files updated."
+
+# Run npx projen commands and ensure everything properly builds
+echo "Compiling..."
+npx projen compile
+echo "Running npx projen..."
+npx projen
+echo "Building..."
+npx projen pre-compile
+echo "Deploying integ test..."
+if ! yes | npx projen integ:kubectl-layer:deploy; then
+  echo "Command failed, but continuing script execution..."  # May fail due to notices, as "destroy" doesn't use --no-notices
+fi
+echo "Checking that the snapshot was updated..."
+if ! npx projen integ:kubectl-layer:assert; then
+  echo "It was not. Updating now..."
+  npx projen integ:kubectl-layer:snapshot
+fi
+echo "Verifying build..."
+if ! yarn build; then
+  echo "The build has failed. Please review the error messages above and determine if it is due to your local environment."
+  read -p "A PR will be created. Please ensure the 'build' test on that passes before merging. Press enter to continue."
+fi
+
+# Remind user to make a new branch in go library as well
+echo "Please visit $(tput setaf 3)https://github.com/cdklabs/awscdk-kubectl-go/branches$(tput sgr0) and create a new branch titled '$(tput setaf 3)kubectl.$1$(tput sgr0)'."
+read -p "Press Enter to continue..."
+
+# Commit changes and push PR
+git add .
+git commit -m "feat: support v1.$1"
+git push origin kubectl-v${1}/main
```

Comment:
did this branch get created at some point? or is that still a manual thing?

Reply:
I believe maintainer will still need to create the base branch in the repo, since the changes get done in their fork. I can update the README to include that, unless there's some command we can use to automate that as well? But there are none that I'm aware of.